shell/gpu/gpu_surface_gl_delegate.h

```diff
@@ -48,6 +48,9 @@ struct GLPresentInfo {
   // The buffer damage refers to the region that needs to be set as damaged
   // within the frame buffer.
   const std::optional<SkIRect>& buffer_damage;
+
+  // For android pipeline TODO(moffatman): comment
+  int64_t vsync_id;
```

Comment:
Please document, here and elsewhere, the properties of the vsync_id and/or link to documentation.

Is it monotonically increasing? What happens if we drop a frame, et cetera.

Reply:
Sure. In general FYI, I believe it is just monotonically increasing. But we are supposed to, and we can, just treat it as a magic token and never modify it or inspect it.

Reply:
(initialize this to something)
```suggestion
  int64_t vsync_id = 0;
```